scripts/cloudbuild.sh

```diff
@@ -17,11 +17,19 @@
 set -e
 
 projectRoot=`dirname $0`/..
+buildProperties=$projectRoot/target/build.properties
+
+# reads a property value from a .properties file
+function read_prop {
+  grep "${1}" $buildProperties | cut -d'=' -f2
+}
+
+# invoke local maven to generate build properties file
+mvn generate-resources
```

Comment:
Try to run the execution directly with `mvn properties:write-project-properties@my-exec`.